src/protvista-uniprot.ts

```diff
@@ -124,7 +101,7 @@ type ProtvistaCategory = {
   'color-range'?: string;
 };
 
-export type DownloadConfig = {
+type DownloadConfig = {
```

Comment:
We do not have download in protvista-uniprot anymore. we can delete it

Reply:
Removed